CMakeLists.txt

```diff
@@ -203,7 +207,9 @@ endif()
 # builds without need for -fno-omit-frame-pointer
 set_source_files_properties("${CMAKE_CURRENT_SOURCE_DIR}/src/openloco/interop/interop.cpp" PROPERTIES COMPILE_FLAGS -fno-omit-frame-pointer)
 
-set(CMAKE_CXX_STANDARD 17)
+if (NOT CMAKE_CXX_STANDARD)
```

Comment:
What's the reason for this change? I would prefer this not be set externally, the project relies on c++17

Reply:
your compiler must not support for C++17

Reply:
yes

Reply:
Where is your `CMAKE_CXX_STANDARD` set then? This is up to the project to set the required standard, not user.